Tests/secrets_white_list.json

```diff
@@ -662,6 +662,8 @@
   },
   "generic_strings":
   [
+    "self.conf_json_validator",
+    "self.IMAGE_MAX_SIZE:",
```

Comment:
Currently works on in lowercase, either lower case it or add .lower() when comparing string against the whitelist.